packages/core/src/services/locale/locale.service.ts

```diff
@@ -117,4 +119,9 @@ export class LocaleService extends Disposable {
 
         return null;
     }
+
+    private _changeHtmlDirectionBaseOnLocale(locale: LocaleType): void {
+        const isRTL = RTL_LOCALES.includes(locale);
+        document.documentElement.setAttribute('dir', isRTL ? 'rtl' : 'ltr');
+    }
```

Comment:
Your contributions are fantastic, thank you so much! 😍 

I believe it's best to leave management of the `dir` attribute on the `documentElement` to developers. Rather than enforcing a specific setting, perhaps Univer could provide guidance or recommendations on best practices? What are your thoughts?

Reply:
@jikkai 
Thank you so much for reviewing my PR ❤

Yeah you're absolutly right, I will make the requested change.